hypervisor/include/arch/x86/timer.h

```diff
@@ -50,6 +50,7 @@ struct hv_timer {
 #define CYCLES_PER_MS	us_to_ticks(1000U)
 
 void udelay(uint32_t us);
+void usleep(uint32_t us);
```

Comment:
This API is used for long duration delay with schedule. So, how about to use ms as parameter? name it as sleep?
Many other software also use sleep with similar meaning

Reply:
void msleep(uint32_t ms) ?